MultiSlider.js

```diff
@@ -140,6 +141,11 @@ export default class MultiSlider extends React.Component {
   };
 
   moveOne = gestureState => {
+    if (this.props.moveLimitOne === this.props.values[0]) {
+      if (gestureState.dx > 0)
```

Comment:
This needs to work with vertical too. Check in the file for a example how to check for dy.

Reply:
it's not working my side please explain more about it how to use and all